values/otomi-console/otomi-console.gotmpl

```diff
@@ -44,3 +44,7 @@ env:
   CORE: '{"services":{{ $v.services | toJson }},"teamConfig":{"services":{{ $v.teamConfig.services | toJson }}}}'
   CORE_VERSION: '{{ $version }}'
   TEAMS: '{{ keys $v.teamConfig.teams | toJson }}'
+
+
+podAnnotations:
+  policies.otomi.io/ignore: "bannedimagetags,pspallowedusers"
```

Comment:
redundant quotes

Reply:
ok, I understand they are redundant but I can;t remove them now. why not focus on something more crucial.